test/api_test.exs

```diff
@@ -216,9 +311,20 @@ defmodule Surface.APITest do
       end)
     end
 
+    test "validate slot as option" do
+      code = "slot cols, as: [:info, {:a, :b}]"
+
+      message =
+        ~r/invalid value for option :as in slot. Expected an atom, got: \[:info, {:a, :b}\]/
+
+      assert_raise(CompileError, message, fn ->
+        eval(code)
+      end)
+    end
```

Comment:
Do you think we need both, this one and this one https://github.com/msaraiva/surface/pull/230/files#diff-6edab3c057af39388f5ff5b56b1d815b3e1935acb8ae7782a7c3be52fd96209fR53-R59 ?

Reply:
No we do not need both. Good catch!